docs/deploy.md

```diff
@@ -9,12 +9,22 @@ The following are the parameters supported by this goal in addition to the [comm
 | Parameter | Description | Required |
 | --------  | ----------- | -------  |
 | appsDirectory | The server's `apps` or `dropins` directory where the application files should be copied. The default value is set to `apps` if the application is defined in the server configuration, otherwise it is set to `dropins`.  | No |
-| stripVersion | Strip artifact version when copying the application to Liberty runtime's application directory. The default value is `false`. | No |
+| copyDependencies | Copies the specified dependencies to the specified locations. Multiple `dependency` elements can be added to the `copyDependencies` configuration. The `location` element can be added to the `copyDependencies` or `dependency` configuration to override the default location, which is ${server.config.dir}/lib/global. | No |
```

Comment:
`${server.config.dir}` is a server config property not a maven property... is this confusing at all ? Should we just spell it out what we mean instead of effectively referring to https://www.ibm.com/support/knowledgecenter/SS7K4U_liberty/com.ibm.websphere.wlp.zseries.doc/ae/rwlp_dirs.html doc ? (Or do we already do require the user to know this is what we're talking about?)

Reply:
I'll reword it.